tests/functional/object/mcg/test_noobaa_secret.py

```diff
@@ -343,7 +351,7 @@ def test_noobaa_secret_deletion_method2(self, teardown_factory, mcg_obj, cleanup
 @bugzilla("2219522")
 @polarion_id("OCS-5205")
 @runs_on_provider
-@tier2
+@acceptance
```

Comment:
I am not sure if this test can be part of acceptance as it is not a functionality specific test. @hnallurv wdyt?

Reply:
@PrasadDesala I think its best to move it to tier1. I moved it to acceptance because its something related to security and impotant that we check. But now i think its well suited for tier1

Reply:
Yes, let's move it to tier1